features/templates.feature

```diff
@@ -0,0 +1,19 @@
+@templates
+Feature: Templates
+
+    Description from screwdriver-cd/screwdriver#470
+
+    Background:
+        Given "calvin" has admin permission to the template
+
+    @ignore
+    Scenario Outline: A template owner wants to validate their template
```

Comment:
You're right. The scenario does state that Calvin has admin permissions, even though we don't enforce it. It's also intentional that we don't describe a scenario where a non-admin user interacts with the endpoint.